packages/entities/entities-shared/src/components/entity-base-config-card/EntityBaseConfigCard.vue

```diff
@@ -187,6 +256,30 @@ const { axiosInstance } = composables.useAxios({
   headers: props.config?.requestHeaders,
 })
 
+const configFormatItems = [
+  {
+    label: 'Structured',
+    value: 'structured',
+    selected: true,
+  },
+  {
+    label: 'JSON',
+    value: 'json',
+  },
+  {
+    label: 'YAML',
+    value: 'yaml',
+  },
+]
+
+const konnectJsonYamlEnabled = computed(() => props.config.app === 'konnect' && props.config.jsonYamlEnabled)
```

Comment:
what is the purpose of computed there?

Reply:
I am looking at this now and think I can use a method call instead of computed here since we don't require caching for this variable. What do you think ?

Reply:
I think computed should be fine since it's evaluating two props